articles/rules/current/index.md

```diff
@@ -44,6 +44,10 @@ A Rule is a function with the following arguments:
 
 * `callback`: a function to send back potentially modified tokens back to Auth0, or an error. Because of the async nature of Node.js, it is important to always call the `callback` function, or else the script will timeout.
 
+::: note
```

Comment:
@mpaktiti I made the changes as requested. Let me know if anything else is needed